osc-service-api/src/main/java/org/osc/core/broker/service/request/AddOrUpdateServiceFunctionChainRequest.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.service.request;
+
+import org.osc.core.broker.service.dto.ServiceFunctionChainDto;
+
+public class AddOrUpdateServiceFunctionChainRequest extends BaseRequest<ServiceFunctionChainDto> {
```

Comment:
you may not need this. servicefuntionchain parent id is the VC id. You can javadoc the ServiceFunctionChainDto